modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -192,8 +199,7 @@ When selected, the dashed line becomes orange (active), and the right working pa
 
 image::1.png[]
 
-To map a column in the data file to a vertex or edge field, first select the row representing the data column in the left table, then click the target field in the right table.
-
+To map a column in the data file to a vertex or edge field, drag and drop a specific data point from the data file to a vertex or edge.
```

Comment:
Here we don't drag and drop a specific data point from the data file to a vertex or edge. We drag and drop from a row representing a data column(csv)/key(json) in the left table to a target field in the right table. The target field is either an attribute of the vertex/edge, a primary id for vertex or a source and target id for an edge.